common/src/main/java/org/apache/celeborn/common/meta/FileInfo.java

```diff
@@ -46,6 +46,7 @@ public class FileInfo {
   // members for MapPartition
   private int bufferSize;
   private int numSubpartitions;
+  private long fileSize;
```

Comment:
Shall we also modify ```PbSerDeUtils.toPbFileInfo``` and '''PbSerDeUtils.fromPbFileInfo''' to add fileSize in it?

Reply:
Yes, it should. Forgot about worker's graceful shutdown.